src/executables/unload_schema.c

```diff
@@ -3934,3 +4045,691 @@ create_filename (const char *output_dirname, const char *output_prefix, const ch
 
   return 0;
 }
+
+static int
+create_filename (const char *output_dirname, const char *output_prefix, const char *middle, const char *suffix,
```

Comment:
As the parameter, I understand 'const char *middle' takes 3 types name;  SCHEMA_NAME, TRIGGER_NAME, INDEX_NAME.

I am a little bit confused because what purpose of 'middle'. 

Would you change the parameter name according to the purpose?
Or Would you add the comment above the create_filename function?

Do we need to make this parameter clear?
I suggest parameter name to be more clear because I think this function is created by this 'middle' parameter. 

I can suggest below;
1. file_type
2. expression
3. infix

If my suggestion is not adequate what you design, could you explain about the design?

Reply:
This function is just a function to generate a file by receiving three inputs, and it is not a function with special significance.
So, create_filename is designed to input three values to create a file in the format of [dbname]_[schema_name]_[object_name].
Since the format of the file can be changed at any time, the parameter was written in an ambiguous way. So the function name is also "create_filename" with no special meaning.

Reply:
So, in your opinion,is parameter name of 'middle' clear to understand?

Reply:
Thank you for your comments.
I confirmed that the meaning of the word was not used correctly.
I would change it to "infix" instead of "middle".

Reply:
@airnet73 
Oh, I understand. 
It seems look so good.